npm/vue/src/index.ts

```diff
@@ -105,9 +144,9 @@ export function mount (
  *  import {mountCallback} from '@cypress/vue'
  *  beforeEach(mountVue(component, options))
```

Comment:
I wonder if this comment should be `beforeEach(mountCallback(component, options))`

I don't see a lot of value in this function at all, I think we should just recommend people do 

```
beforeEach(() => {
  mount(Blah, opts)
})
```

🤔